### ISSUE TITLE
Visual bugs on arm32 phones

### ISSUE THREAD

**Issue by sounddrill31** [Sangwan5688/BlackHole#544]
![logs.txt](https://github.com/Sangwan5688/BlackHole/files/11096091/logs.txt)

Disclaimer: I am running a custom rom but I do not have any other arm 32 phones to test this on. 

The app is very buggy visually on my phone with arm32(happens on beta(1.15.7), stable(1.15.6), both armeabi and universal(the visual bugs appear different between the these). On other phones, it's a very good app with fast loading, and good downloads.

I've seen this problem with revanced manager and a couple of other apps in the past as well on this phone.

My specs: 

Oneplus One with snapdragon 801
LineageOS 18.1 official but rooted

**Comment by Sangwan5688** [comment]
Please share screenshots for the same

**Comment by sounddrill31** [comment]
https://user-images.githubusercontent.com/84176052/229338528-1389c146-18cf-4d1f-8a84-2bd094856148.mp4



A screenshot might not demonstrate properly, this video attempts to. 

Device: Same, Oneplus one running lineageOS 18.1

Version: 1.15.7

**Comment by Sangwan5688** [comment]
Strange, but this doesn't look like an issue with the app. Might be some issue with your device. Is this happening with this app only?

**Comment by sounddrill31** [comment]
> Strange, but this doesn't look like an issue with the app. Might be some issue with your device. Is this happening with this app only?

Makes sense if it is my device only, revanced manager also gave similar issues. On the other hand, I've used many other flutter apps like neobackup without any issues of any sort.
